sys/include/net/zep.h

```diff
@@ -7,12 +7,16 @@
  */
 
 /**
- * @defgroup    net_zep  ZEP header definitions
+ * @defgroup    net_zep  ZigBee Encapsulation Protocol
  * @ingroup     net
+ * @brief       Provides definitions for the ZigBee Encapsulation Protocol
+ *
+ * @see drivers_socket_zep
```

Comment:
This doesn't resolve (see above). And I'm not sure if this link makes sense. `socket_zep` is just one implementation of ZEP. Others might be provided. Would this list need to be amended then? This should be avoided IMHO.

Reply:
I also tried to to add socket_zep as a subgroup of net_zep (and drivers_netdev) but the doxygen breadcrumb doesn't render well in this case.

Reply:
Ok, finally I removed this line